core/src/main/module.gwt.xml

```diff
@@ -22,7 +22,8 @@
   -->
 
 <module>
-    <inherits name="com.google.common.base.Base"/>
-    <inherits name="com.google.common.collect.Collect"/>
+    <inherits name="elemental2.core.Core"/>
+    <inherits name="elemental2.dom.Dom"/>
+    <inherits name="elemental2.webstorage.WebStorage"/>
```

Comment:
webstorage include dom and core, but I think adding the 3 modules is more explicit, meaning that we are actually using each module, "typifying" events and elements of each one.